app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
Oh yeah, sorry, I meant locking down the event type and object-type if we move it under `/services` to always apply to `update_service`.

I don't really see the benefits of keeping it generic across audit event types to be honest. There's no work on the roadmap that I'm aware of that would require this and as I've said above chances are the flow would be different anyway.

At the same time, my main practical point is about how easy it is to gather information about what's going on. Reordering the URL doesn't really help with that. If you take that draft-services endpoint for example (which I believe is largely unused) - you'd look at the service_id in the draft service and search for log events that contain it. I don't really see an easy way to do this with the proposed audit-events endpoints - you have to find related events until you can search the logs.